src/common/java/com/minelittlepony/gui/GameGui.java

```diff
@@ -0,0 +1,40 @@
+package com.minelittlepony.gui;
+
+import org.apache.commons.lang3.text.WordUtils;
+
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.client.gui.GuiScreen;
+import net.minecraft.client.resources.I18n;
+
+public abstract class GameGui extends GuiScreen {
+
+    @Override
+    protected void actionPerformed(GuiButton button) {
+        if (button instanceof IActionable) {
+            ((IActionable)button).perform();
+        }
+    }
+
+    protected static String format(String string, Object... pars) {
+        return I18n.format(string, pars);
```

Comment:
Becaise otherwise we have to write imports for I18n in all of out classes.

Reply:
Why would that matter?

Reply:
I'd like to minimise the usage of obfuscated classes, and hiding this means we can just write `format("options.title")` instead of `I18n.format("options.title")`. Or just `"options.title"` if you're using one of the Gui elements that handle translatoons transparently.

Reply:
Again, why would that matter? Obfuscated classes work fine.

Reply:
*until their names change

Reply:
Name changes still don't matter. The names are handled automatically on build.

Reply:
I've had to deal with MCP names changing multiple times in the past. Besides that I really want to hide translation calls as much as possible.

Reply:
Either way, there's no need for it to be static. Things that aren't subclasses can just call I18n.format directly.